apps/codeforafrica/src/payload/globals/Publication/index.js

```diff
@@ -0,0 +1,20 @@
+import PostsTab from "./PostsTab";
+
+const Publication = {
+  slug: "publication",
+  label: "Publication",
+  access: {
+    read: () => true,
+  },
+  admin: {
+    group: "Website",
```

Comment:
```suggestion
    group: "Settings",
```

`admin.group` in `Users` and `Site` needs to be updated as well.